### ISSUE TITLE
glDeleteTextures error under Linux

### ISSUE THREAD

**Issue by mickp** [MicronOxford/cockpit#358]
Enable a camera.
Snap an image.
Disable the camera.

```
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/wx/core.py", line 3240, in <lambda>
    lambda event: event.callable(*event.args, **event.kw) )
  File "/home/map/cockpit/cockpit/gui/imageViewer/viewCanvas.py", line 203, in clear
    self.tiles[i][j].wipe()
  File "/home/map/cockpit/cockpit/gui/imageViewer/image.py", line 225, in wipe
    GL.glDeleteTextures(self.textureID)
  File "/usr/lib/python3.6/site-packages/PyOpenGL-3.1.1a1-py3.6.egg/OpenGL/latebind.py", line 61, in __call__
    return self.wrapperFunction( self.baseFunction, *args, **named )
  File "/usr/lib/python3.6/site-packages/PyOpenGL-3.1.1a1-py3.6.egg/OpenGL/GL/exceptional.py", line 59, in glDeleteTextures
    ptr = arrays.GLuintArray.asArray( size )
  File "/usr/lib/python3.6/site-packages/PyOpenGL-3.1.1a1-py3.6.egg/OpenGL/arrays/arraydatatype.py", line 149, in asArray
    return cls.getHandler(value).asArray( value, typeCode or cls.typeConstant )
  File "/usr/lib/python3.6/site-packages/PyOpenGL-3.1.1a1-py3.6.egg/OpenGL/arrays/arraydatatype.py", line 53, in __call__
    typ.__module__, type.__name__, repr(value)[:50]
TypeError: No array-type handler for type numpy.type (value: 1) registered
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/wx/core.py", line 3240, in <lambda>
    lambda event: event.callable(*event.args, **event.kw) )
  File "/home/map/cockpit/cockpit/gui/imageViewer/viewCanvas.py", line 203, in clear
    self.tiles[i][j].wipe()
  File "/home/map/cockpit/cockpit/gui/imageViewer/image.py", line 225, in wipe
    GL.glDeleteTextures(self.textureID)
  File "/usr/lib/python3.6/site-packages/PyOpenGL-3.1.1a1-py3.6.egg/OpenGL/latebind.py", line 61, in __call__
    return self.wrapperFunction( self.baseFunction, *args, **named )
  File "/usr/lib/python3.6/site-packages/PyOpenGL-3.1.1a1-py3.6.egg/OpenGL/GL/exceptional.py", line 59, in glDeleteTextures
    ptr = arrays.GLuintArray.asArray( size )
  File "/usr/lib/python3.6/site-packages/PyOpenGL-3.1.1a1-py3.6.egg/OpenGL/arrays/arraydatatype.py", line 149, in asArray
    return cls.getHandler(value).asArray( value, typeCode or cls.typeConstant )
  File "/usr/lib/python3.6/site-packages/PyOpenGL-3.1.1a1-py3.6.egg/OpenGL/arrays/arraydatatype.py", line 53, in __call__
    typ.__module__, type.__name__, repr(value)[:50]
TypeError: No array-type handler for type numpy.type (value: 1) registered
```

**Comment by mickp** [comment]
cockpit.gui.imageView.image.Image.wipe is getting called twice, so on the second call it tries to delete a texture with a handle of None. Easy workaround is to add if self.textureId is not None, but why is it called twice?

**Comment by mickp** [comment]
Because viewpanel.ViewPanel.disable explicitly calls it twice.

**Comment by mickp** [comment]
OK - so now only called once, but still see this error on the first call.

**Comment by mickp** [comment]
glDeleteTextures expects an array of texture names.
Fixed by 4f181762d82175ffa2226d655466d1d6ecf263d7.
